src/components/Course/index.js

```diff
@@ -19,16 +20,51 @@ function Course() {
     return <Loader pageLoader={true} />;
   }
 
+  const handleChange = (e) => {
+    e.preventDefault();
+    setSearch(e.target.value);
+  };
+
+  let filteredCourse;
+  if (data) {
+    filteredCourse = data.allCourses.filter((names) => {
+      return names.name.toLowerCase().includes(search.toLowerCase());
+    });
+  }
+
   return (
-    <div className="ng-course">
-      <CourseList
-        list={get(data, "enrolledCourses")}
-        title="Aap in courses mein enrolled hai"
-      />
-      <CourseList
-        list={get(data, "allCourses")}
-        title="Aap yeh courses mein enroll kar skte hai"
-      />
+    <div>
+      <div className="search">
+        <input
+          type="text"
+          className="search-term"
+          placeholder="What are you looking for?"
```

Comment:
Perhaps the better placeholder could be.. `Search for a course. ex. Python`
What do you think? cc @pysaquib

Reply:
Makes sense

Reply:
done